x2b_default_config.toml

```diff
@@ -0,0 +1,16 @@
+[slurm-args]
+time = "04:00:00"
+mem = 16000
+nodes = 1
+cpus-per-task = 2
+job-name = "xnat2bids"
+output = "/gpfs/data/bnc/scratch/logs/%J.txt"
+
+[xnat2bids-args]
+session = "XNAT_E00080"
+bids_root = "/gpfs/data/bnc/shared/bids-export/"
+version = ""
+bids_root_dir=""
```

Comment:
Are these really supposed to be empty? Do you even need the `bids_root_dir`? Or can we make it relative to the one above and only have one of them specified in the config file